lua/org-bullets.lua

```diff
@@ -72,20 +73,22 @@ local markers = {
   end,
   -- Checkboxes [x]
   expr = function(str, conf)
-    local text = { str, "OrgTODO" }
     local symbols = conf.symbols.checkboxes
     if str:match("X") then
-      text = symbols.done
-    elseif str:match("%-") then
-      text = symbols.cancelled
-    else
-      text = symbols.todo
+      return { { "[", "OrgTSCheckboxChecked" }, symbols.done, { "]", "OrgTSCheckboxChecked" } }
+    elseif str:match("-") then
+      return {
+        { "[", "OrgTSCheckboxHalfChecked" },
+        symbols.half,
+        { "]", "OrgTSCheckboxHalfChecked" },
```

Comment:
I can't see the `CheckboxHalfChecked` highlight group? Is this an official highlight group?

Reply:
![org_hl](https://user-images.githubusercontent.com/82431193/173689711-7be712fe-d9d2-4336-bdeb-0f1959dc3f80.png)

Reply:
@lvim-tech that doesn't really answer the question 😅 I don't know where that screenshot is from is that the orgmode plugin repo?